pkg/collector/parser.go

```diff
@@ -56,8 +56,10 @@ func ParseTransport(t *transport.Transport, line string) {
 		case "receive-buffer":
 			t.ReceiveBuffer = val
 		case "peer":
+		  if t.CurrentPeer.Number != nil {
+		    t.Peers = append(t.Peers, t.CurrentPeer)
+		  }
```

Comment:
```suggestion
			if t.CurrentPeer.Number != nil {
				 t.Peers = append(t.Peers, t.CurrentPeer)
			}
```

yadada whitespace vs tabs